Source/Scene/Expression.js

```diff
@@ -343,6 +369,34 @@ define([
         return feature.getProperty(this._value);
     };
 
+    function checkFeature (ast) {
+        return (ast._value === 'feature');
+    }
+
+    // PERFORMANCE_IDEA: Determine if parent property needs to be computed before runtime
```

Comment:
Currently, as long as an expression evaluates to a string literal it should be possible. So `${foo[${memberName}]}` would work, except I do not believe we currently parse variables inside of variables.